src/client/components/ClientState.tsx

```diff
@@ -1,7 +1,9 @@
 import React, { createContext, FunctionComponent } from 'react';
 import { ClientState } from '@/shared/model/ClientState';
 
-export const ClientStateContext = createContext({});
+export const ClientStateContext = createContext({
+  clientHosts: { idapiBaseUrl: '' },
```

Comment:
if you've already added the defaults to server/models/Express.ts then i'm not sure you need to set the default here?

Reply:
I have to set it because client hosts field is not optional
```
 Type '{}' is not assignable to type 'ClientState'.
```